edgedetect_windows.go

```diff
@@ -9,7 +9,7 @@ import (
 func defaultBrowserIsEdge() bool {
 	k, err := registry.OpenKey(registry.CLASSES_ROOT, `ActivatableClasses\Package\DefaultBrowser_NOPUBLISHERID\Server\DefaultBrowserServer`, registry.QUERY_VALUE)
 	if err != nil {
-		log.Errorf("Error reading DefaultBrowserServer registry key: %v", err)
+		log.Debugf("Error reading DefaultBrowserServer registry key: %v", err)
```

Comment:
I changed this to debug since it's actually a somewhat expected condition at this point and not an error.